### PR TITLE
8461 non expressive variable name

### LINKED ISSUE #8461: Non expressive variable name on `transformer.py`
**Description of Problem**:
Issue: On file `...\utils\tensorflow\transformer.py` in the line `566` has a variable with a non expressive name `i`.

**Overview of the Solution**:
Change the name from `i` to something more expressive.

### PR DESCRIPTION
**Pull request by joaoCeilandia**
**Proposed changes**:
-fix #8461 

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: rasa/utils/tensorflow/transformer.py [hunk 1]
```diff
@@ -563,8 +563,8 @@ def __init__(
         self._layer_norm = tf.keras.layers.LayerNormalization(epsilon=1e-6)
 
     def _get_angles(self) -> np.ndarray:
-        i = np.arange(self.units)[np.newaxis, :]
-        return 1 / np.power(10000, (2 * (i // 2)) / np.float32(self.units))
+        array_2D = np.arange(self.units)[np.newaxis, :]
```
Comment (ArjaanBuijk) [inline rasa/utils/tensorflow/transformer.py:566]: please use all lowercase: `array_2d`

File: rasa/utils/tensorflow/transformer.py [hunk 2]
```diff
@@ -563,8 +563,8 @@ def __init__(
...
+        return 1 / np.power(10000, (2 * (array_2D // 2)) / np.float32(self.units))
```
Comment (ArjaanBuijk) [inline rasa/utils/tensorflow/transformer.py:567]: please use all lowercase: `array_2d`

**Review by ArjaanBuijk — CHANGES_REQUESTED** [review]
To be pep8 compliant, please use all lowercase for the new variable name.

File: rasa/utils/tensorflow/transformer.py (hunk 2 above)
Comment (joaoCeilandia) [reply rasa/utils/tensorflow/transformer.py]: ok

File: rasa/utils/tensorflow/transformer.py (hunk 1 above)
Comment (joaoCeilandia) [reply rasa/utils/tensorflow/transformer.py]: ok

**Review by ArjaanBuijk — APPROVED** [review]
Thank you!

**Comment by ArjaanBuijk** [comment]
@joaoCeilandia ,
All is ready to be merged, but your branch is out-of-date with the base branch, so I cannot merge it.
Can you please update the branch in your fork, or make your branch writeable for me?

**Comment by joaoCeilandia** [comment]
Has no conflicts with the base branch now.  I don't know if i can make my branch writeable for you because it is from a class in my college. @ArjaanBuijk

**Comment by ArjaanBuijk** [comment]
@joaoCeilandia ,
Can you bring your branch up-to-date with the base branch again?

sorry it has to be done this way, but if you are not able to make your branch writeable for me, you will have to update it manually again.

Things are added quickly, and your PR is in the queue to be merged in, but it needs to be squeezed in there.

**Comment by ArjaanBuijk** [comment]
@joaoCeilandia ,
thanks for the PR!

**Pull request merged**